apps/chat/app/globals.css

```
@@ -350,7 +350,7 @@
 }
 
 @utility glow-blue { box-shadow: var(--ag-shadow-glow-blue); }
-@utility glow-green { box-shadow: var(--ag-shadow-glow-green); }
+@utility glow-accent { box-shadow: var(--ag-shadow-glow-accent); }
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)